src/wallets/wallets.js

```diff
@@ -314,27 +344,41 @@ function sameSender(sender1, sender2) {
   return JSON.stringify(sender1.publicKey) == JSON.stringify(sender2.publicKey);
 }
 
+async function signSet(signer, senderAddressObj, txnarray) {
+  const senderAddressObj = algosdk.decodeAddress(info.address);
```

Comment:
@ElianaTroper came across an 'identifier already declared' error here, is this constant declaration needed (and renamed?) since senderAddressObj is passed to the function? Or should the constant stay and parameter removed